tests/c_languages/CPPCheckBearTest.py

```diff
@@ -15,20 +15,36 @@ def get_absolute_test_path(file):
 
 
 @generate_skip_decorator(CPPCheckBear)
-class CPPCheckBearTest(LocalBearTestHelper):
+class CPPCheckBearTest(unittest.TestCase):
```

Comment:
Why not keeping `LocalBearTestHelper`? Then you can also leave the tests from the earlier commit :+1:

Reply:
How I can keep `LocalBearTestHelper` as I am writing test for a `GlobalBear`

Reply:
ahhh yeah good point sorry :D

Reply:
I have made the required changes.
I'm trying to write `GlobalBearTestHelper` because now I have some knowledge about how `LocalBearTestHelper` is working ;) Should I go for it or will it be too hard for me now.